app/components/star-icon/calendar-icon.tsx

```diff
@@ -0,0 +1,27 @@
+import { observer } from "mobx-react-lite"
+import { Image, ImageStyle, TouchableOpacity, ViewStyle } from "react-native"
+const calenderImage = require("../../../assets/calendar.png")
+import React from "react"
+
+export const CalendarIcon = observer(function CalendarIcon(props: {
+  style?: ViewStyle
+  onPress: () => void
+}) {
+  const { onPress, style } = props
+  const ICON_STYLE: ImageStyle = {
+    width: 27,
+    height: 25,
+    resizeMode: "contain",
+    tintColor: "lightgrey",
+    opacity: 0.9,
+  }
+  const CONTAINER: ViewStyle = {
+    justifyContent: "center",
+  }
+
+  return (
+    <TouchableOpacity onPress={ onPress } style={ [CONTAINER, style] }>
+      <Image source={ calenderImage } style={ [ICON_STYLE] } />
```

Comment:
The icon isn't aligned properly and looks kinda old, we should change it to a different one. What do you think @guytepper?

Reply:
this is a free icon I've found. Do you have any other free design system to take the icon from ?

Reply:
I saw that you added a new icon, but it's not aligned like the favorites button. The calendar icon is more to the left.